lib/populate.py

```diff
@@ -105,32 +105,60 @@ def populate_all(
         json_root,
         is_valid_stream_name,
         ):
-    streams = get_streams(client)
-    ind = {}
-    for s in (s for s in streams if is_valid_stream_name(s['name'])):
-        print(s['name'])
-        topics = safe_request(client.get_stream_topics, s['stream_id'])['topics']
-        nind = {'id': s['stream_id'], 'latest_id':0}
-        tpmap = {}
+    all_streams = get_streams(client)
+
+    streams = [s for s in all_streams if is_valid_stream_name(s['name'])]
+
+    streams_data = {}
+
+    for s in streams:
```

Comment:
Maybe this could be `for s in all_streams`, then have a check
```python
if not is_valid_stream_name(s['name']):
    continue
```

Reply:
Yeah, I thought about that, but I try to avoid `continue` in big loops when I can, as it's nice to know that there's only one execution path through a loop.